test/jdl/jdl-importer.spec.js

```diff
@@ -57,6 +57,7 @@ describe('JDLImporter', () => {
                     ],
                     relationships: [
                         {
+                            otherEntityField: 'region',
```

Comment:
Shouldn't it be `id`?

Reply:
https://github.com/jhipster/generator-jhipster/blob/da8e8bae66c9a94260305b8233129274032c6912/test/jdl/test-files/big_sample.jdl#L85
It’s correct isn’t it?

Reply:
`region` (the inner one) should be the displayed field. There are two possibilities here: the JDL is wrong or the `otherEntityField` is weirder than I though. Or both.
It shouldn't have been `id` but `area`, right?

Reply:
Tell me If I'm getting confused :laughing: I hate relationships

Reply:
It’s correct, `region` is the displayed name which is `otherEntityField`.
And `area` is the `relationshipName`.

Reply:
Ah! Thanks :)